query-builder/src/test/java/com/datastax/dse/driver/internal/querybuilder/DependencyCheckTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.dse.driver.internal.querybuilder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Paths;
+import java.util.Properties;
+import org.junit.Test;
+
+public class DependencyCheckTest {
+
+  @Test
+  public void should_generate_deps_txt() throws IOException {
```

Comment:
maybe we can extract this method as `assertFIleInResourcesExists(Path path)` to a core module, and call it from every `DependencyCheckTest` to reduce duplication?

Reply:
I thought about it, but the only way to do it would be one of:
1) publish the core module's test jar (so it can be used by querybuilder and mapper modules)
2) move the utility method to core module's regular source code
I don't think we want to start publishing the test jar. I can try to move this into a `util` package in the core and see what that looks like.

Reply:
We do publish the core module's test jar. I think you can factor the test in a parent class so that you only need to override the path. That way you don't have to put the check in a production class (which I'm not a fan of).

Reply:
My mistake. I'll refactor it.